packages/Radio/src/Radio.styles.js

```diff
@@ -0,0 +1,196 @@
+  return hasLabel ? `${toInt(radioSize) + toInt(tokens.space)}px` : radioSize;
+};
+
+const smallRadioSize = tokens.radio.sizeSm;
+const mediumRadioSize = tokens.radio.sizeMd;
+const largeRadioSize = tokens.radio.sizeLg;
+
+const getHalfSizeCss = sizeCss => `${toInt(sizeCss) / 2}px`;
+const smallRadioHalfSize = getHalfSizeCss(smallRadioSize);
+const mediumRadioHalfSize = getHalfSizeCss(mediumRadioSize);
+const largeRadioHalfSize = getHalfSizeCss(largeRadioSize);
+
+const styles = {
+  [ShirtSizes.SMALL]: {
+    baseFontSize: {
+      fontSize: `${fontSizeValue(-1)}px`,
+    },
+    radioStyles: {
+      height: smallRadioSize,
+      width: smallRadioSize,
+      borderRadius: smallRadioHalfSize,
+    },
+    radioIconBackgroundStyles: {
+      borderRadius: "4px",
+      height: "8px",
+      top: "4px",
+      width: "8px",
+    },
+    radioIconStyles: {
+      fontSize: `${fontSizeValue(-4)}px`,
+      height: smallRadioSize,
+      left: smallRadioHalfSize,
+    },
+    labelStyles: hasLabel => {
+      return {
+        minHeight: smallRadioSize,
+        padding: `0 0 0 ${getLabelLeftPadding(smallRadioSize, hasLabel)}`,
+      };
+    },
+  },
+  [ShirtSizes.MEDIUM]: {
+    baseFontSize: {
+      fontSize: `${fontSizeValue()}px`,
+    },
+    radioStyles: {
+      height: mediumRadioSize,
+      width: mediumRadioSize,
+      borderRadius: mediumRadioHalfSize,
+    },
+    radioIconBackgroundStyles: {
+      borderRadius: "6px",
+      height: "10px",
+      top: "5px",
+      width: "10px",
+    },
+    radioIconStyles: {
+      fontSize: `${fontSizeValue(-2)}px`,
+      height: mediumRadioSize,
+      left: mediumRadioHalfSize,
+    },
+    labelStyles: hasLabel => {
+      return {
+        minHeight: mediumRadioSize,
+        padding: `1px 0 0 ${getLabelLeftPadding(mediumRadioSize, hasLabel)}`,
+      };
+    },
+  },
+  [ShirtSizes.LARGE]: {
+    baseFontSize: {
+      fontSize: `${fontSizeValue()}px`,
+    },
+    radioStyles: {
+      height: largeRadioSize,
+      width: largeRadioSize,
+      borderRadius: largeRadioHalfSize,
+    },
+    radioIconBackgroundStyles: {
+      borderRadius: "6px",
+      height: "12px",
+      top: "6px",
+      width: "12px",
+    },
+    radioIconStyles: {
+      fontSize: `${fontSizeValue()}px`,
+      height: largeRadioSize,
+      left: largeRadioHalfSize,
+    },
+    labelStyles: hasLabel => {
+      return {
+        minHeight: largeRadioSize,
+        padding: `3px 0 0 ${getLabelLeftPadding(largeRadioSize, hasLabel)}`,
+      };
+    },
+  },
+};
+
+const radioStyles = css`
+  ${boxSizingStyles};
+  ${({ size }) => styles[size].baseFontSize};
+  line-height: ${({ hasLabel }) => (hasLabel ? lineHeightValue(-1) : "0")};
+  margin: ${tokens.space} 0;
+  position: relative;
+
+  input[type="radio"] {
+    ${visuallyHidden};
+
+    &:focus + label::before {
+      box-shadow: ${tokens.highlight.active.noBorder.boxShadow};
+      border-color: ${tokens.highlight.active.noBorder.borderColor};
```

Comment:
This doesn't seem to be taking affect, the border is still black when focussed:

![Screen Shot 2019-11-25 at 1 26 50 PM](https://user-images.githubusercontent.com/14944896/69579416-47edf480-0f87-11ea-9f16-d2f571b3e4ee.png)